floodviz/utils.py

```diff
@@ -0,0 +1,90 @@
+import requests
+
+"""
+    Takes a valid url and a dictionary of expected column header strings as keys (values do not matter).
+    parse_rdb then requests data from the service at url, and extracts data matching any header values.
+     
+    Data is returned as a list of dict objects, where each object represents a data point. 
+
+"""
+
+
+def parse_rdb(url, header_values):
+    content = None
+    try:
+        r = requests.get(url)
+    except requests.exceptions.RequestException as e:
+        print('- Bad URL -')
+    else:
+        if r.status_code is 200:
+            content = r.text.splitlines()
+    if content:
```

Comment:
Isn't this (line 21) essentially the same as `if r.status_code is 200:`, since `content` is never set if the status is not 200?